presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -135,6 +136,8 @@ public void testResetSession()
     @Test
     public void testCreateTable()
     {
+        featuresToTest(TestedFeature.CREATE_TABLE);
```

Comment:
This test method also includes DROP TABLE. I already said `TestedFeature.DROP_TABLE` shouldn't exist, but if it remains, it should be included here.